### ISSUE TITLE
评估自己训练好的检测模型，评估结果较差，该怎么优化？？

### ISSUE THREAD

**Issue by xiaohua1027** [PaddlePaddle/PaddleOCR#4244]
Global:
  debug: false
  use_gpu: true
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\output\ch_db_mv3_new
  save_epoch_step: 100
  eval_batch_step:
  - 2000
  - 1000
  cal_metric_during_train: false
  pretrained_model: C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\pretrain_models\ch_ppocr_mobile_v2.0_det_train\best_accuracy
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: null
  save_res_path: C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\output\det_db\predicts_db.txt
  distributed: false
Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
    disable_se: true
  Neck:
    name: DBFPN
    out_channels: 96
  Head:
    name: DBHead
    k: 50
Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 0
PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5
Metric:
  name: DetMetric
  main_indicator: hmean
Train:
  dataset:
    name: SimpleDataSet
    data_dir: C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\train_data\
    label_file_list:
    - C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\train_data\train_label.txt
    ratio_list:
    - 1.0
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 960
        - 960
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 8
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\train_data\
    label_file_list:
    - C:\Users\sugon\Desktop\dsh_paddleocr\PaddleOCR-release-2.1\train_data\test_label.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest: null
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2

**Comment by xiaohua1027** [comment]
![G4BB$QZF@IUFR)E)GQA2U F](https://user-images.githubusercontent.com/75288803/135806429-bdb357eb-03ef-4022-a022-5aa9a75fcf68.png)

**Comment by xiaohua1027** [comment]
迭代次数太高的话训练会要很长时间。。。
我的数据集都是自然场景下的，训练数据集有6433个，测试集1601个
识别模型也是同样的问题，训练数据集有148001个，测试集64022个

**Comment by ioracion** [comment]
1.把“你要的效果”图片跟“实际效果”图片po出来看看，光看参数没啥意义
2.换resnet模型，mobile效果确实不太好
3.你要串联的话，在检测模型训练好后，最好用检测模型标出的文本框当作识别模型的训练数据，不能用“你自己标的完美”的图片去训练，除非你保证检测模型能标的跟你一模一样

**Comment by 18362890185** [comment]
> 你要串联的话，在检测模型训练好后，最好用检测模型标出的文本框当作识别模型的训练数据，不能用“你自己标的完美”的图片去训练，除非你保证检测模型能标的跟你一模一样

关于用检测模型的文本框作为训练集，如何操作呢？我用的DB模型检测定位，并没有发现定位抠出的区域 麻烦告知
